easy-random-core/src/main/java/org/jeasy/random/util/ReflectionUtils.java

```diff
@@ -133,6 +137,27 @@ public Object invoke(final Object proxy, final Method method, final Object[] arg
      * @throws IllegalAccessException if the property cannot be set
      */
     public static void setProperty(final Object object, final Field field, final Object value) throws IllegalAccessException, InvocationTargetException {
+        //TODO it seems that 'propertyUtilsBean.setProperty(..)' does not work for Maps
```

Comment:
Seems to be supported for me, according to this https://github.com/apache/commons-beanutils/blob/commons-beanutils-1.9.4/src/main/java/org/apache/commons/beanutils/PropertyUtilsBean.java#L1908.

Reply:
@benas  i think some related test failed but maybe it was annother problem